cime_config/namelist_definition_drv.xml

```diff
@@ -165,12 +185,17 @@
   <entry id="drv_restart_pointer" modify_via_xml="DRV_RESTART_POINTER">
     <type>char</type>
     <category>expdef</category>
+    <default_value>UNSET</default_value>
```

Comment:
This default_value seems redundant with the value settings below and can probably be removed.